packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 5.4.0 - Dec 12, 2023
+
+This minor version update of Teams Toolkit includes new features and bug fixes based on customers feedback. The new features include the integrated Adaptive Cards Previewer, the Teams app test tool for Teams Bot app and new project templates for AI Assistant Bot, and refreshed Sample Gallery look. We previously shared these incremental changes in the prerelease version and through a series of blog posts:
+
+- [October Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-update-october-2023): Search sample app, new Contoso Retail Dashboard sample app, Teams Toolkit CLI improvement.
+- [November Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-for-visual-studio-code-update-november-2023): Brand-new AI Assistant Bot App template, Teams app test tool, integrated Adaptive Card Previewer, Refreshed look for Sample Gallery etc.
+
+We've listened to your feedback and included these additional new features, enhancements, and bug fixes to this release.
+
+New features:
+
+- AI Assistant Bot App template is powered by the Teams AI library and the OpenAI Assistants API, this template demonstrates how to build an intelligent chatbot within Teams.
+  ![AI-assistant-bot](https://github.com/OfficeDev/TeamsFx/assets/113089977/13877e7b-cffb-4717-9d1e-587478a43e5d)
+- Teams App Test Tool empowers developers to debug, test, and iterate on the Teams Bot app design within a web-based chat environment.
+  ![Teams app test tool](https://github.com/OfficeDev/TeamsFx/assets/113089977/7f6bcf56-27b3-4a7b-a160-64ab847efd92)
+- Preview and edit Adaptive Cards effortlessly with the integrated Adaptive Card Previewer directly within Teams Toolkit.
+  ![Adaptive Card Previwer](https://github.com/OfficeDev/TeamsFx/assets/113089977/8062f237-d640-45b2-a7b2-81a0a4b0aa10)
+- Sample Gallery new design offers grid view and list view, making it easy to navigate among samples. You can also search samples or filter samples by type, capability, and programming language.
```

Comment:
Navigate quickly through samples using list or grid views. You can also search samples or filter samples by type, capability, and programming language.